tests/playbooks/tests_infiniband.yml

```
@@ -2,6 +2,8 @@
 ---
 - name: Play for testing infiniband device
   hosts: all
+  vars:
+    interface: ib0
```

```suggestion
    interface: "{{ if lookup('env', 'INFINIBAND_INTERFACE_NAME')
      then lookup('env', 'INFINIBAND_INTERFACE_NAME')
      else 'ib0' }}"
```
Then the name can be set in various ways.  The most convenient is setting env. var. - in most test frameworks we use, it is easy to add an env. var. to the test environment (e.g. 1minutetip, beaker, etc.) e.g.
```bash
INFINIBAND_INTERFACE_NAME=ibxyz runtest.sh ....
```
you can also use `ansible-playbook -e interface=ibxyz` but then `interface` is used by many network tests, so you would need to be careful to only use it for this test.

We should never require the modification of a test playbook.

---

Then, I agree to add a variable to the test environment. let me prepare an update.